### PR TITLE
issue #8: Pure CSS responsive Fork me on GitHub ribbon

### PR DESCRIPTION
**Pull request by rngadam**
screen shot 2015-07-13 at 2 19 38 pm
screen shot 2015-07-13 at 2 19 46 pm

### CODE REVIEW

**Comment by lionello** [comment]
Please change in all html files.

**Comment by rngadam** [comment]
done

File: bouncer.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    top: 60px;
+    right: -60px;
+    transform: rotate(45deg);
+    -webkit-transform: rotate(45deg);
+    -ms-transform: rotate(45deg);
+    -moz-transform: rotate(45deg);
+    -o-transform: rotate(45deg);
+    box-shadow: 4px 4px 10px rgba(0, 0, 0, 0.8);
+  }
+}
```
Comment (lionello) [inline bouncer.css]: nl

**Comment by rngadam** [comment]
nl added

**Pull request merged**